### PR TITLE
chore: merge last develop changes

### PR DESCRIPTION
**Pull request by obdulia-losantos**

### CODE REVIEW

**Review by shawnsarwar — DISMISSED** [review]
The Travis failure in the integration tests looks like a race condition in the tests. I restarted it. :crossed_fingers: I can try to address it in dev tomorrow.

File: test-aether-integration-module/test/__init__.py [hunk 1]
```diff
@@ -110,17 +110,21 @@ def entities(client, schemadecorators):  # noqa: F811
...
+def generate_entities(realm_client, mappingset):  # noqa: F811
+    def fn(realm):
+        _client = realm_client(realm)
+        payloads = iter(fixtures.get_submission_payloads())
+        entities = []
+        for i in range(FORMS_TO_SUBMIT):
+            Submission = _client.get_model('Submission')
+            submission = Submission(payload=next(payloads), mappingset=mappingset.id)
+            instance = _client.submissions.create(data=submission)
+            sleep(2)
```
Comment (obdulia-losantos) [inline test-aether-integration-module/test/__init__.py:122]: @shawnsarwar  this does the trick :grimacing:

Comment (shawnsarwar) [reply test-aether-integration-module/test/__init__.py:122]: :+1:  Makes sense.

**Comment by shawnsarwar** [comment]
This was one of the errors that came up quite a lot while I was working with this last week:
https://travis-ci.com/github/eHealthAfrica/aether/jobs/362919227

**Review by shawnsarwar — APPROVED** [review]

**Pull request merged**